src/main/webapp/user_profile_style.css

```diff
@@ -149,15 +149,15 @@ th, td {
 	color: white;
 }
 
-.delete-task:hover {
+.delete-task:hover, .abandon-task:hover {
 	background-color: red;
 }
 
-.delete-task:hover i {
+.delete-task:hover i, .abandon-task:hover i {
 	color: white;
 }
 
-#createTaskModal {
+#createTaskModal, #editTaskModal {
```

Comment:
Instead of slowly building up a list of IDs, let's make this a class and apply it to the elements. Same for the next several rules that change in this PR.

Reply:
Sure. I let the two modal wrappers belong to the same class `modalWrapper` in the new commit. Similarly, I let the two modals, close buttons of the two modals, and two task tables have their own class respectively. Then I modify the css file to accommodate the change in the new commit.